components/admin_console/workspace-optimization/dashboard.scss

```diff
@@ -43,6 +43,45 @@
                         line-height: 20px;
                     }
                 }
+
+                &__extraContent {
+                    display: flex;
+                    flex-direction: row;
+                    align-items: flex-end;
+                    margin-left: auto;
+
+                    button {
+                        width: 120px;
+                        height: 24px;
+                        justify-content: space-around;
+                        border: none !important;
+                        box-shadow: none !important;
```

Comment:
Any way we can avoid using `!important` in this file?

Reply:
they were not needed actually, good catch.